python/samples/04.ai.f.dataSource.azureOpenAI/src/bot.py

```diff
@@ -11,97 +11,74 @@
 
 from botbuilder.core import MemoryStorage, TurnContext
 from teams import Application, ApplicationOptions, TeamsAdapter
+from teams.ai import AIOptions
 from teams.ai.models import AzureOpenAIModelOptions, OpenAIModel
 from teams.ai.planners import ActionPlanner, ActionPlannerOptions
-from teams.ai.prompts import PromptManager, PromptManagerOptions, PromptTemplate
-from teams.state import TurnState
-from teams.ai import AIOptions
-from azure.identity import get_bearer_token_provider, DefaultAzureCredential
-from pathlib import Path
+from teams.ai.prompts import PromptManager, PromptManagerOptions
 
 from config import Config
 from state import AppTurnState
 
 config = Config()
 
 if config.AZURE_OPENAI_KEY is None:
-    raise RuntimeError(
-        "Missing environment variables - please check that AZURE_OPENAI_KEY is set."
-    )
+    raise RuntimeError("Missing environment variables - please check that AZURE_OPENAI_KEY is set.")
 
 # Create AI components
 model: OpenAIModel
 
 model = OpenAIModel(
     AzureOpenAIModelOptions(
+        # Uncomment desired authentication method
         # api_key=config.AZURE_OPENAI_KEY,
-        azure_ad_token_provider=get_bearer_token_provider(DefaultAzureCredential(), 'https://cognitiveservices.azure.com/.default'),
+        # azure_ad_token_provider=get_bearer_token_provider(DefaultAzureCredential(), 'https://cognitiveservices.azure.com/.default'),
         default_model="gpt-4o",
         api_version="2024-02-15-preview",
         endpoint=config.AZURE_OPENAI_ENDPOINT,
     )
 )
 
-prompts = PromptManager(PromptManagerOptions(prompts_folder=f"{os.path.dirname(os.path.abspath(__file__))}/prompts"))
-
-# gets prompt template and adds data source config
-async def get_default_prompt(context: TurnContext, state: TurnState, planner: ActionPlanner) -> PromptTemplate:
-    prompt = await prompts.get_prompt("chat")
-    
-    prompt.config.completion.model = 'gpt-4o'
-    
-    if config.AZURE_SEARCH_ENDPOINT:
```

Comment:
sorry, to clarify- where did we end up moving this to?

Reply:
i just deleted them since that's what was done in JS sample